server/apiController.js

```diff
@@ -118,3 +120,76 @@ module.exports.getTrailer = (req, res) => {
     .then(results => res.send(results.data.items[0]))
     .catch(err => res.sendStatus(404));
 };
+
+module.exports.getSearchAutoComplete = (req, res) => {
```

Comment:
Having your autocomplete involve not one but two requests means when it comes time to deploy, your autocomplete may well be too slow to be of any use.  I'd suggest getting as much of this functionality onto the front-end as possible.  Other strategies can be employed to help speed up the process, too:  caching a user's previous searches locally, pre-loading a number of popular/recent titles, etc.